src/rapidata/rapidata_client/selection/capped_selection.py

```diff
@@ -21,5 +21,5 @@ def to_model(self):
                 CappedSelectionSelectionsInner(selection.to_model())
                 for selection in self.selections
             ],
-            max_rapids=self.max_rapids,
+            maxRapids=self.max_rapids,
```

Comment:
Why this change?

Reply:
it's something i encountered. the backend changed it and i guess we never did realize it was broken